### PR TITLE
[vagrant] dependencies, error redirection, secure urls 

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: setup/setup_vagrant.sh [hunk 1]
```diff
@@ -3,12 +3,13 @@
 # It's only tested with ubuntu 14.04
 set -e
 
-REFUGE_PATH=/vagrant/refugerestrooms
+REFUGE_PATH=/vagrant
```
Comment (dunn) [inline setup/setup_vagrant.sh]: This should have been changed alongside 30301d259cbfe16eea1d71ae329a6804ab9150e1.

File: setup/setup_vagrant.sh [hunk 2]
```diff
@@ -3,12 +3,13 @@
...
 
 # required packages
 declare -A packages
-packages=( 
+packages=(
   ["git"]="=1:1.9.1-1"
+  ["libreadline-dev"]=""
```
Comment (dunn) [inline setup/setup_vagrant.sh]: Building Ruby from source failed for me without installing readline first.

**Comment by tkwidmer** [comment]
Leaving this one up to @mi-wood since I dont know vagrant at all.

**Comment by mi-wood** [comment]
Lgtm thanks!

**Pull request merged**